### DISCUSSION TITLE
Formily能支持微信小程序吗

### DISCUSSION THREAD

**Discussion by zhaixinhua** [alibaba/formily#2076] [Q&A 2.x] [answered]
在PC端进行动态表单生成，在小程序上进行动态表单的渲染，实现动态表单的多端适配，请问哪些功能需要阉割，哪个功能实现不了，以及小程序怎么进行适配，哪位大神做过求指导。

**Comment by cuiyiworld** [comment]
你好，请问你实现了吗？我正好也有这样的需求。

**Comment by janryWang** [comment, marked as answer]
用 taro 不是很轻松就支持的？

**Reply by galan99** [reply]
taro怎么支持的，ui用nutui-react-taro，跑的时候直接报错

**Comment by TTQ99** [comment]
小程序端不支持动态执行js代码。 在JSONSchema中书写表达式， 需要用到 new Function。 在小程序端用不了

**Reply by yiyunwan** [reply]
不用表达式就可以了我理解是
